openapi/platform/platform.yaml

```diff
@@ -3068,6 +3068,12 @@ components:
                       - monthly
                       - yearly
                     description: The billing frequency
+                  isPackage:
+                    type: boolean
+                    description: Indicates if this lineitem is a package instead of a product or an addon.
```

Comment:
It might be good to mention that the fields are readonly so don't need to be sent when creating orders.